chord/chord.go

```diff
@@ -88,9 +93,11 @@ func NewChordNode(listener net.Listener) (*ChordNode, error) {
 		sync.RWMutex{},
```

Comment:
did you add rwmutex to Node struct?

Reply:
I know that unmutexed chordNode.node causes data races, but it's not changed after the initialization, so we should be safe. However, I am interested in how you manage to put mutex in inside a struct that is generated by someone else and which doesn't have a mutex inside its type?

Reply:
I will do the rest of the review after I work through the code in my editor. For now, there is enough info for you to reply to :D

Reply:
> did you add rwmutex to Node struct?

Nope

Reply:
> did you add rwmutex to Node struct?

No, this line not modified/added by me.